config/environments/test.rb

```diff
@@ -1,42 +1,67 @@
 Rails.application.configure do
```

Comment:
What actually changed in this file within the context of this work?

Reply:
I copied and pasted the `development.rb` contents as this meant that the test users were generated when running in the test scope. I could try and narrow down the required part if you would like.